plugins/fileservices/fileservices.cpp

```diff
@@ -2516,6 +2516,7 @@ FILESERVICES_API char * FILESERVICES_CALL fsGetEspURL(const char *username, cons
     Owned<IConstEnvironment> daliEnv = openDaliEnvironment();
     Owned<IPropertyTree> env = getEnvironmentTree(daliEnv);
 
+    StringBuffer espURL;
     if (env.get())
     {
         StringBuffer espURL;
```

Comment:
it is confusing to still have this variable with the same name in a nested scope.  Either remove, or use strdup("") as the default return.

Reply:
I think I intended to remove the nested one. Let me look